[sonnen] Update to API V2 of vendor and add PowerMeter (#14589)
openhab/openhab-addons: bundles/org.openhab.binding.sonnen/README.md

The binding for sonnen communicates with a sonnen battery.
More information about the sonnen battery can be found [here](https://sonnen.de/).

> The binding supports now the old deprecated V1 from sonnen as well as V2 which requires a Authentication token.

Reviewer: ```suggestion
The binding supports the old deprecated V1 from sonnen as well as V2 which requires an authentication token.
```

Author: ok

> More infromation about the V2 Api can be found http://LOCAL-SYSTEM-IP/api/doc.html

Reviewer: ```suggestion
More information about the V2 API can be found `http://LOCAL-SYSTEM-IP/api/doc.html`
```

Author: ok

> More infromation about the V2 API can be found `http://LOCAL-SONNENBATTERY-SYSTEM-IP/api/doc.html`

Reviewer: ```suggestion
More information about the V2 API can be found at `http://LOCAL-SONNENBATTERY-SYSTEM-IP/api/doc.html`
```

## Thing Configuration

Only the parameter `hostIP` is required; this is the IP address of the sonnen battery in your local network.

> If you want to use the V2 API of the vendor sonnen which supports more channels you need to provide the `authToken`.

Reviewer: ```suggestion
If you want to use the V2 API, which supports more channels, you need to provide the `authToken`.
```

> If you want to use the V2 API of the vendor sonnen which supports more channels you need to provide the `Authentication Token`.

Reviewer: Actual parameter name:
```suggestion
If you want to use the V2 API of the vendor sonnen which supports more channels you need to provide the `authToken`.
```

It might also be an idea to reformat this into a nice table describing both parameters.

Author: ok

| flowConsumptionProductionState | Switch        | read   | Indicates if there is a current flow from Solar Production towards Consumption          |
| flowGridBatteryState           | Switch        | read   | Indicates if there is a current flow from Grid towards Battery                          |
| flowProductionBatteryState     | Switch        | read   | Indicates if there is a current flow from Production towards Battery                    |
| energyImportedStateProduction  | Number:Energy | read   | Indicates the imported kWh Production                                                   |
| energyExportedStateProduction  | Number:Energy | read   | Indicates the exported kWh Production                                                   |

> | energyImportedStateConsumption | Number:Energy | read   | Indicates the imported kWh Production                                                   |
> | energyExportedStateConsumption | Number:Energy | read   | Indicates the exported kWh Production                                                   |

Reviewer: ```suggestion
| energyImportedStateConsumption | Number:Energy | read   | Indicates the imported kWh Consumption                                                  |
| energyExportedStateConsumption | Number:Energy | read   | Indicates the exported kWh Consumption                                                  |
```

Author: ok
